Pictionary

cvglass opened the pull request:
pictionary has webapp functionality (hopefully). Has main frame, scoreboard, canvas. Reducers will add teams and track scores. Still need to have team avatar render and update the server.

cvglass committed:
pictionary frame and first reducer

cvglass committed:
pictionary frame and first reducer

cvglass committed:
pictionary show drawing frame

cvglass committed:
changed ip address

cvglass committed:
pictionary has webapp functionality (hopefully). Has main frame, scoreboard, canvas. Reducers will add teams and track scores. Still need to have team avatar render and update the server.

ayoung09 commented on browser/components/pictionary/PictionaryScoreboard.js:
```diff
@@ -0,0 +1,45 @@
+    })
+  }
+
+  componentDidUnmount() {
```
should be componentWillUnmount

ayoung09 commented on browser/components/pictionary/PictionaryScoreboard.js:
```diff
@@ -0,0 +1,45 @@
+    const avatarCoordinates = this.props.team.portrait;
+    const scores = this.props.scores[this.props.team.name]
+    return (
+      <div className="user-thumbnail">
```
We might want to change the className on this so that we can format it differently than the individual Drawkward thumbnails later. Maybe "team-thumbnail"?

cvglass committed:
Update PictionaryShowDrawing.js

cvglass committed:
fixed componentDidUnmount to componentWillUnmount

cvglass committed:
changed css tag

I put the DrawkwardUserThumbnail as a placeholder, but I still need to review the component to see if it's reusable. If not, I'll create a new one.

cvglass committed:
Merge branch 'master' into pictionary
